docs/tutorials/try-pre-built-images/install-on-a-device/install-on-renesas.md

```diff
@@ -0,0 +1,236 @@
+
+To configure Wi-Fi, press the cursor up key until the wireless device (such as _wlan0_) is selected and press **Enter**. You will see a small menu and you need to select **Edit Wi-Fi**.
+
+![Edit Wi-Fi](https://assets.ubuntu.com/v1/f4889452-f4889452487e4f2eb052ad7410d078a8c0bddd06.png)
+
+After selecting _Edit Wi-Fi_, you will see the network interface configuration panel for Wi-Fi. If you know the name of the Wi-Fi network you wish to connect to, it can be entered directly, or cursor down to _Choose a visible network_ and select a network from a list of those that have been detected.
+
+![Enter password](https://assets.ubuntu.com/v1/04bb9874-bf6ddac76e5fdc34bfdcf353e035c4e1d679d5a4_2_690x385.png)
+
+Finally, enter the Wi-Fi password and select **Save** to complete the configuration. You will be returned to the previous menu and your device will attempt to connect to the network. If successful, you will see its IP address to the right of DHCPv4.
+
+![DHCP](https://assets.ubuntu.com/v1/9a7c81a5-9a7c81a5d6b5202dcb8f222649339a761dd40c45.png)
+
+For advanced network configuration, such as setting a static IP address, select the wireless device again, and choose edit IPv4 from the menu.
+
+When you've finished configuring your network settings, select **Done** and press enter on the _Network connections_ page to move on to the final step.
+
+#### Ethernet
+
+If an Ethernet cable is connected to your device, a network connection will attempt to be automatically negotiated and, if this is successful, you will see an IP address for the device after the DHCPv4 entry in the _Network connections_ page. In this case, you don't need to do anything further:
+
+![Choose eth0](https://assets.ubuntu.com/v1/74586cbe-74586cbe2090bb63c3e7017907011293dc950458.png)
+
+To configure an Ethernet connection manually, select the eth0 device and select _Edit IPv4_ from the small menu that appears. By default, the network device will be configured to use _Automatic (DHCP)_, which is why the connection attempts to automatically configure itself. Press **Enter** to reveal two further options, _Manual_ and _Disabled_:
+
+![Select Manual](https://assets.ubuntu.com/v1/01acc192-01acc19205a26a77cbb96021a47e36f0a3f9eff6.png)
+
+Selecting _Manual_ will allow you to configure your Ethernet connection manually by entering values for your subnet mask value (using CIDR `xx.xx.xx.xx/yy` notation), the static IP address of your device, the network gateway, and the name servers you wish to use:
+
+![Edit eth0 IPv4 configuration](https://assets.ubuntu.com/v1/ad78cf8a-ad78cf8a153e449632f6033c2359e69737039415.png)
+
+Select **Save** to apply those changes and for the connection to be attempted. You can now proceed to the next step by pressing **Done**.
+
+### Add Ubuntu SSO details
+
+This is the final step before completing the Ubuntu Core setup and you simply need to enter the email address linked to your Ubuntu SSH account.
```

Comment:
I might be wrong, but it's Ubuntu One or Ubuntu account, not "Ubuntu SSH account"